src/topology/instances/nnreal.lean

```diff
@@ -219,4 +219,11 @@ begin
   exact tendsto_tsum_compl_at_top_zero (λ (a : α), (f a : ℝ))
 end
 
+/-- `x ↦ x ^ n` as an order isomorphism of `ℝ≥0`. -/
+def pow_order_iso (n : ℕ) (hn : n ≠ 0) : ℝ≥0 ≃o ℝ≥0 :=
```

Comment:
Maybe `n` can be implicit here?

Reply:
Then it will be pretty printed as `pow_order_iso _`.